cni/cmd/kuma-cni/injector_windows.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+import "errors"
+
+func Inject(netns string, intermediateConfig *IntermediateConfig) error {
+	return errors.New("only implemented on linux")
+}
```

Comment:
You can use build flags for target envs if you want .e.g at the start of the file`// +build aix darwin dragonfly freebsd js,wasm linux nacl netbsd openbsd solaris`

See: https://www.digitalocean.com/community/tutorials/building-go-applications-for-different-operating-systems-and-architectures

Reply:
What more does that give us vs the suffix approach? Also what is the desired outcome for trying to build on windows / darwin - build error / runtime error?

Reply:
I'm leaving this as it is, you need me to change it then please reach out via DM / open an issue.